### PR TITLE
RoosterJS dependencies update

### PR DESCRIPTION
**Pull request by vhuseinova-msft**
# What

- update RoosterJS dependencies
- exported DefaultSanitizers from roosterjs and removed local copy
- added announcements when bulleted and numbered lists button are clicked
- added announcements when new bulleted and numbered lists items are added 

# Why

# How Tested

# Process & policy checklist

- [ ] I have updated the project documentation to reflect my changes if necessary.
- [ ] I have read the [CONTRIBUTING](https://github.com/Azure/communication-ui-library/blob/main/CONTRIBUTING.md) documentation.

**Is this a breaking change?**

- [ ] This change causes current functionality to break.

### CODE REVIEW

File: packages/react-components/src/components/RichTextEditor/Toolbar/RichTextToolbar.tsx [hunk 1]
```diff
@@ -110,14 +110,23 @@ export const RichTextToolbar = (props: RichTextToolbarProps): JSX.Element => {
       icon: 'RichTextBulletListButtonIcon',
       onClick: () => {
         plugin.onToolbarButtonClick((editor) => {
+          // check the format state to see if the bulleted list is already applied
+          const isBullet = formatState?.isBullet;
           toggleBullet(editor);
+          // the bulleted list was added
+          if (!isBullet) {
+            setTimeout(() => {
+              // a small delay and polite aria live are needed for MacOS VoiceOver to announce the change
```
Comment (palatter) [inline packages/react-components/src/components/RichTextEditor/Toolbar/RichTextToolbar.tsx:119]: Is this delay something we have done for other announcements?

File: packages/storybook8/stories/Composites/ChatComposite/Docs.mdx [hunk 2]
```diff
@@ -79,8 +78,8 @@ we support hiding the `Topic Header`:
 ## Participant Pane
 
 <SingleLineBetaBanner />
-You can also show a `Participant Pane` to the right hand side of the chat that has support for removing users. The Participant
-pane is a `beta` feature. Currently it does not support mobile views and is subject to breaking changes.
+You can also show a `Participant Pane` to the right hand side of the chat that has support for removing users. The
```
Comment (palatter) [inline packages/storybook8/stories/Composites/ChatComposite/Docs.mdx:81]: Should 'The' be on the line below?

**Review by Leah-Xia-Microsoft — APPROVED** [review]

File: packages/react-components/src/components/RichTextEditor/RichTextEditor.tsx [hunk 3]
```diff
@@ -152,14 +159,6 @@ export const RichTextEditor = React.forwardRef<RichTextEditorComponentRef, RichT
     return new RichTextToolbarPlugin();
   }, []);
 
-  const isDarkThemedValue = useMemo(() => {
-    return isDarkThemed(theme);
-  }, [theme]);
-
-  useEffect(() => {
-    editor.current?.setDarkModeState(isDarkThemedValue);
```
Comment (vhuseinova-msft) [inline packages/react-components/src/components/RichTextEditor/RichTextEditor.tsx:160]: we don't need this anymore as editor creation `useEffect` already depends on theme and will be updated when theme is changed

File: packages/react-components/src/components/RichTextEditor/Toolbar/RichTextToolbar.tsx (hunk 1 above)
Comment (vhuseinova-msft) [reply packages/react-components/src/components/RichTextEditor/Toolbar/RichTextToolbar.tsx:119]: THere are some places. In this case we change focus when toggle list functions are called, adding the delay helps MacOS to queue everything correctly

By the way, Windows works without it as per RoosterJS engineer feedback

File: packages/storybook8/stories/Composites/ChatComposite/Docs.mdx (hunk 2 above)
Comment (vhuseinova-msft) [reply packages/storybook8/stories/Composites/ChatComposite/Docs.mdx:81]: I'm not sure why I have these changes, it looks like linter has applied some styling changes

Comment (vhuseinova-msft) [reply packages/storybook8/stories/Composites/ChatComposite/Docs.mdx:81]: The text in Storybook looks okay though
image

File: packages/react-components/src/components/RichTextEditor/Toolbar/RichTextToolbar.tsx (hunk 1 above)
Comment (vhuseinova-msft) [reply packages/react-components/src/components/RichTextEditor/Toolbar/RichTextToolbar.tsx:119]: also, the delay value is 50 milliseconds which is pretty small amount

**Review by palatter — APPROVED** [review]

**Pull request merged**